### ISSUE TITLE
[Balance Issue] Bronze Blend crafting only gives 1 per 4 ingot/ore.

### ISSUE THREAD

**Issue by v3xed-v0id** [Khazoda/bronze#24]
As per the title, i have found a balance issue with the mod, as bronze blend only gives 1 per every 4 ore/ingot, when it should be giving 4 blend as you put in 4 ore/ingot into the crafting recipe to make it in the first place, this makes it very underpowered IMO and i would appreciate it if it could give 4 blend in the craft, thank you.

**Comment by v3xed-v0id** [comment]
its also just pretty unrealistic that you would get less ingot in than what you put into the crafting recipe to begin with IMO.

**Comment by Khazoda** [comment]
Hi, I'll break down why bronze blend to bronze ingot is a 1:1 recipe:
- Every bronze ingot is made from **one** tin ore
- Bronze is better than iron stats wise for weapons/tools/armour, so should therefore be harder to get
  - Tin ore is slightly rarer than iron ore, but not by much
- Copper is a plentiful resource that doesn't have many uses outside of decorative blocks in vanilla. Having the recipe for 1 bronze ingot be 1 tin + 3 copper gives that copper, that you likely have lying around a lot, a new use.
- In real life, the ratio of tin to copper to create bronze is about 1:4. This mod's recipe for bronze blend is as close to that as possible (1:3) while still fitting in the 2x2 crafting table so you can make bronze blend on the go.

If the realism isn't enough for you, I encourage you to pretend the lost mass is impurities lost during the crafting & smelting process.

TLDR: bronze is definitely not underpowered

**Comment by v3xed-v0id** [comment]
> Hi, I'll break down why bronze blend to bronze ingot is a 1:1 recipe:
> 
> * Every bronze ingot is made from **one** tin ore
> * Bronze is better than iron stats wise for weapons/tools/armour, so should therefore be harder to get
>   
>   * Tin ore is slightly rarer than iron ore, but not by much
> * Copper is a plentiful resource that doesn't have many uses outside of decorative blocks in vanilla. Having the recipe for 1 bronze ingot be 1 tin + 3 copper gives that copper, that you likely have lying around a lot, a new use.
> * In real life, the ratio of tin to copper to create bronze is about 1:4. This mod's recipe for bronze blend is as close to that as possible (1:3) while still fitting in the 2x2 crafting table so you can make bronze blend on the go.
> 
> If the realism isn't enough for you, I encourage you to pretend the lost mass is impurities lost during the crafting & smelting process.
> 
> TLDR: bronze is definitely not underpowered

the realism aspect still doesnt make sense, IRL you dont lose that much mass and in game 1 ore = 1 ingot, the 4:1 ratio is dilution of the metal, you would still get 5 ingots from the ratio assuming it was 4 copper ingot and 1 tin ingot, its just dilution and not lost mass

i know what you are trying to say here, but this "explanation" clearly shows that you lack knowledge on IRL metalsmithing (which i understand your lack of knowledge, most people do not know IRL metalsmithing that well)

**Comment by v3xed-v0id** [comment]
> Hi, I'll break down why bronze blend to bronze ingot is a 1:1 recipe:
> 
> * Every bronze ingot is made from **one** tin ore
> * Bronze is better than iron stats wise for weapons/tools/armour, so should therefore be harder to get
>   
>   * Tin ore is slightly rarer than iron ore, but not by much
> * Copper is a plentiful resource that doesn't have many uses outside of decorative blocks in vanilla. Having the recipe for 1 bronze ingot be 1 tin + 3 copper gives that copper, that you likely have lying around a lot, a new use.
> * In real life, the ratio of tin to copper to create bronze is about 1:4. This mod's recipe for bronze blend is as close to that as possible (1:3) while still fitting in the 2x2 crafting table so you can make bronze blend on the go.
> 
> If the realism isn't enough for you, I encourage you to pretend the lost mass is impurities lost during the crafting & smelting process.
> 
> TLDR: bronze is definitely not underpowered

as a compromise, could you add a toggle in the mod settings to enable crafting 4 bronze blend from 3 copper + 1 tin?

**Comment by v3xed-v0id** [comment]
> the realism aspect still doesnt make sense, IRL you dont lose that much mass and in game 1 ore = 1 ingot, the 4:1 ratio is dilution of the metal, you would still get 5 ingots from the ratio assuming it was 4 copper ingot and 1 tin ingot, its just dilution and not lost mass
> 
> I do understand the balance argument from your perspective, however.

also im sorry for being a bit rude in this message initially, i edited it to remove the rude part, im not doing well mentally rn, hope you didnt take it personally, sorry about it

**Comment by Khazoda** [comment]
@v3xed-v0id 
That's alright don't worry 😊
Recipes in Minecraft are actually fully datapackable, so you can change the recipe if you want to!

I quite like this tool which makes it super easy to make custom recipes: https://crafting.thedestruc7i0n.ca/
The item ids from the mod you're gonna want are:

bronze:bronze_blend
bronze:raw_tin
bronze:bronze_ingot (if you want to modify the blend smelting recipe)

**Comment by v3xed-v0id** [comment]
> @v3xed-v0id 
> That's alright don't worry 😊
> Recipes in Minecraft are actually fully datapackable, so you can change the recipe if you want to!
> 
> I quite like this tool which makes it super easy to make custom recipes: https://crafting.thedestruc7i0n.ca/
> The item ids from the mod you're gonna want are:
> 
> bronze:bronze_blend
> bronze:raw_tin
> bronze:bronze_ingot (if you want to modify the blend smelting recipe)
> 
> 

Thank you, i will take a look at it :3
